pom.xml

```diff
@@ -37,10 +37,6 @@
     </scm>
 
     <repositories>
-        <repository>
-            <id>vaadin-addons</id>
-            <url>http://maven.vaadin.com/vaadin-addons</url>
-        </repository>
         <repository>
             <id>vaadin-prerelease</id>
```

Comment:
Do we beed that one still?

Reply:
Yep, I think vaadin-prerelease repository shouldn't be there either.

Reply:
We don't need it now, but we will need to add it again when we start with V11 alphas, done.